io.openems.edge.meter.janitza/readme.adoc

```diff
@@ -1,7 +1,7 @@
-= Janitza UMG 96RM-E Meter
+= JANITZA Meters UMG96RME | UMG604
```

Comment:
The official writing seems to be "Janitza" and "UMG 604" https://www.janitza.de/umg-604-pro.html